app/controllers/mechanics_controller.rb

```diff
@@ -0,0 +1,12 @@
+class MechanicsController < ApplicationController
+  
+  def show
+    @mechanic = Mechanic.find(params[:id])
+  end
+
+  def update
```

Comment:
You won't need an `update` action here if you ended up creating a join controller `RideMechanics`. In there you would have a `create` action. The reason being is that we are not truly updating a mechanic. That would be if we needed to update an attribute on a mechanic. What we are doing is creating an association between and existing mechanic and an existing ride.